planet/src/main/webapp/itinerary.html

```diff
@@ -87,4 +87,23 @@
             </div>
         </div>
     </body>
+    <script>
+        function initAutocomplete() {
+            let startAddress = document.getElementById('starting-address');
+            let eventAddress = document.getElementById('add-event-address');
+
+            let options = {
+                types: ['geocode']
+            };
+            autocomplete = new google.maps.places.Autocomplete(startAddress,options); 
+            autocomplete = new google.maps.places.Autocomplete(eventAddress,options);
+            autocomplete.setFields(['address_component']);
+            autocomplete.addListener('place_changed', fillAddress);
+        }
+
+        function fillAddress() {
+            let place = autocomplete.getPlace();
+        }
+    </script>
```

Comment:
Why did you add these two functions in the html file?

Reply:
I was calling them from the JS file and I kept getting the error "initAutocomplete does not exist" and I tried a bunch of solutions such as moving the api call in the html file and adding async defer but in the end they didn't work

Reply:
So this is the solution I went with

Reply:
Which js file did you have them in?

Reply:
Itinerary.js

Reply:
This is the error: 
![Screenshot 2020-07-22 at 11 52 33 AM](https://user-images.githubusercontent.com/47041119/88238032-a79fbb00-cc4e-11ea-881a-e2d73aa2597b.png)

Reply:
itinerary.js is a module. functions in a module are not added to the global namespace by default.

If you add this function to the global namespace in itinerary.js, what is the behaviour? See lines 18-28 of that file for examples

Reply:
I got the same error when I was implementing the get place details function (PR #35 ), even after I declared it as a global namespace. So I chose to load the google maps js file synchronously, and call initMap at body onload, which solves the problem

Reply:
Okay! I implemented this in the new commit! thanks